projects/solayer-restaking/index.js

```diff
@@ -0,0 +1,34 @@
+const { sumTokens2, getConnection, } = require("../helper/solana");
+const { PublicKey } = require("@solana/web3.js");
+const ADDRESSES = require('../helper/coreAssets.json')
+
+async function tvl(api) {
+  const connection = getConnection();
+
+  // add SOL staking
+  const stakeAccount = await connection.getAccountInfo(new PublicKey('po1osKDWYF9oiVEGmzKA4eTs8eMveFRMox3bUKazGN2'))
+  api.add(ADDRESSES.solana.SOL, Number(stakeAccount.data.readBigUint64LE(258)))
+
+  // get LST details
+  const data = await connection.getProgramAccounts(new PublicKey('sSo1iU21jBrU9VaJ8PJib1MtorefUV4fzC9GURa2KNn'), {
+    filters: [{ dataSize: 74, },],
+  })
+  const tokensAndOwners = data.map((i) => {
+    const offset = 8
+    const lstMint = new PublicKey(i.account.data.slice(offset + 0, offset + 32));
+    return [lstMint.toString(), i.pubkey.toString()]
```

Comment:
is there a way to compute tokenAccount from this? fetching value by token, owner is more expensive. 

Also, can you use the old folder name, folder names are hard coded in config, changing it here breaks our system